content/blog/2020-06-03-how-learnr-saved-the-day/index.markdown

```diff
@@ -0,0 +1,126 @@
+
+I was also worried about finding a "fun" health dataset, but talking about diabetes, cancer, or some other complex disease did not sound fun to me at all. Knowing about the unpopularity of the session in the previous year, I was starting to feel quite desperate. I believed my session was going to mirror the [bored to death classroom scenes](https://www.youtube.com/watch?v=Yk3-Vb7CH-o) from the infamous Ferris Bueller's Day Off movie.
+
+<img src="bored_classroom.jpg" width="80%" style="display: block; margin: auto;" />
+
+
+
+Luckily, I attended [Kenia Wiedemann's talk](https://www.user2019.fr/static/pres/t258353.pdf) at useR!2019. [Kenia](https://concord.org/about/staff/kenia-wiedemann/) is involved with a research project called [CodeR4MATH](https://concord.org/our-work/research-projects/computing-with-r/), which aims to integrate R programming in high school math classes. While her curriculum also included multiple sessions throughout the semester, she talked about the amazing learnr package and all the cool things you can do with it. We also got a chance to chat about teaching R to teenagers and here some of the important points I remember from our conversation:    
+  
+1. You have to minimize any unnecessary distraction. Using RStudio Cloud or RStudio IDE in this setting might not be the best choice, as there are too many things you need to clarify before getting to the actual point. This is why learnr is perfect for this setting, you can limit what they can see and do very easily, without making it too rigid or boring.
+
+1. Most teenagers are quite self-involved, you have to find a dataset that they can relate to -- if you want to get even *some* of their attention.   
+
+1. Do not be discouraged if a session doesn't go very well. Some days, they might not even feel like listening to you, just because of the way you dress up or the class they had before you.   
+
+I have to admit, at this point I was genuinely terrified and still didn't know what talk about. But I knew the tool I had to use: learnr!
+
+## And learnr saves the day!
+
+### Easy setup 
+
+Beyond its pedagogical advantages, learnr is also a clean and easy way to deal with installation or subscription issues that come with dozens of laptops operating on a Windows. In my case, university was providing all the laptops for the session and it was up to me to set them up. Since we had internet connection available during the session, I certainly did not want to install R locally and was planning on using RStudio Cloud. Great, but then you have to then setup separate RStudio Cloud accounts for each machine, which includes setting up different email accounts as well (which we actually did and ended up with dozens of Gmail accounts 😱). 
```

Comment:
- "running Windows" instead of "operating on a Windows"
- "the university" instead of "university"
- "an internet connection" instead of "internet connection"
- "That's great in theory" rather than "Great"